KnownIssues.md

```diff
@@ -19,15 +19,15 @@ If you are still having issues try using a **VPN**, or changing your **DNS provi
 | `1.1.1.1` | `1.0.0.1` | or  | `8.8.8.8` | `8.8.4.4` |
 |---------|---------|-----|---------|---------|
 
-- Script doesn't run/PowerShell crashes:
-  1. Press Windows Key+X and select 'PowerShell (Admin)' (Windows 10) or 'Windows Terminal (Admin)' (Windows 11)
-  2. Run:
-  ```
-  Set-ExecutionPolicy Unrestricted -Scope Process -Force
-  ```
-  3. Run:
-  ```
-  irm christitus.com/win | iex
+- Script doesn't run/PowerShell crashes:	
+  1. Press Windows Key+X and select 'PowerShell (Admin)' (Windows 10) or 'Windows Terminal (Admin)' (Windows 11)	
+  2. Run:	
+  ```	
+  Set-ExecutionPolicy Unrestricted -Scope Process -Force	
+  ```	
+  3. Run:	
+  ```	
+  irm christitus.com/win | iex	
```

Comment:
```suggestion
- Script doesn't run/PowerShell crashes:
  1. Press Windows Key+X and select 'PowerShell (Admin)' (Windows 10) or 'Windows Terminal (Admin)' (Windows 11)
  2. Run:
  ```
  Set-ExecutionPolicy Unrestricted -Scope Process -Force
  ```
  3. Run:
  ```
  irm christitus.com/win | iex
```
There're trailing whitespace characters in these lines @MyDrift-user

Reply:
Thanks @og-mrk for catching the trailing whitespaces, I've just copied and pasted some stuff and didn't notice it.

It is fixed now.